src/Event/ValueObjects/EventStatus.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\ValueObjects;
+
+use Broadway\Serializer\SerializableInterface;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language;
+use InvalidArgumentException;
+
+final class EventStatus implements SerializableInterface
+{
+    /**
+     * @var EventStatusType
+     */
+    private $eventStatusType;
+
+    /**
+     * @var EventStatusReason[]
+     */
+    private $eventStatusReasons;
+
+    public function __construct(EventStatusType $eventStatusType, array $eventStatusReasons)
+    {
+        $this->ensureTranslationsAreUnique($eventStatusReasons);
```

Comment:
Nit: We usually use `guard` I believe, but if you prefer `ensure` that's also okay. It's readable which is the most important. Just wanted to point it out as an FYI.